cmd/abigen/main.go

```diff
@@ -86,28 +85,18 @@ func init() {
 		excFlag,
 		pkgFlag,
 		outFlag,
-		langFlag,
 		aliasFlag,
+		v2Flag,
 	}
-	app.Action = abigen
+	app.Action = generate
 }
 
-func abigen(c *cli.Context) error {
+func generate(c *cli.Context) error {
 	utils.CheckExclusive(c, abiFlag, jsonFlag) // Only one source can be selected.
 
 	if c.String(pkgFlag.Name) == "" {
 		utils.Fatalf("No destination package specified (--pkg)")
 	}
-	if c.String(abiFlag.Name) == "" && c.String(jsonFlag.Name) == "" {
```

Comment:
I think it's still required?
It doesn't make sense to run the generation without input ...